### ISSUE TITLE
Motions import does not work on Linux AppImage

### ISSUE THREAD

**Issue by lucapas70** [arkottke/strata#16]
From version 0.6 the import of motions does not work. I tried on Ubuntu 18.04 and 20.04.
Thank you for this great program.

**Comment by arkottke** [comment]
Does it work in later versions?

**Comment by lucapas70** [comment]
> Does it work in later versions? - Albert

No, version 0.5 works, but from 0.6 onwards I can't import motions. Among other things, version 0.8 crashes by performing the calculation (I always talk about the appimage on Ubuntu 20.04)

**Comment by arkottke** [comment]
My apologies for the delay, I will try to get to this issue this week.

**Comment by arkottke** [comment]
@lucapas70 I finally got around to looking at this. The AppImage works for me if I use `/` for path separators. The current example uses `\` because it was originally written on Windows.

**Comment by lucapas70** [comment]
As visible in the attached screen, in the path of the asci file it is used / as a separator. Once I entered parameters I clicked Apply and in fact, as you can see, the values turn blue. However the Plots tab does not light up and clicking on OK nothing happens. I also attach the accelerograms in question, thanks.
![Schermata da 2020-07-25 08-40-43](https://user-images.githubusercontent.com/65540901/88450931-0cc3fe00-ce53-11ea-9650-099c417a8a6b.png)
[Input_ASCOLI_PICENO_475yrs.zip](https://github.com/arkottke/strata/files/4975012/Input_ASCOLI_PICENO_475yrs.zip)

**Comment by arkottke** [comment]
This is because Strata only knows about the PEER AT2 format of time series. You have to manually load other formats.

**Comment by lucapas70** [comment]
> 
> 
> This is because Strata only knows about the PEER AT2 format of time series. You have to manually load other formats.

But then why does the same file uploaded to Strata for Windows work?
